src/Components/Cards/SpecifyModelQuantityCard/SpecifyModelQuantityCard.js

```diff
@@ -0,0 +1,100 @@
+//#region Import Neccessary Dependencies
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+//#endregion
+
+// Import Stylings
+import './SpecifyModelQuantityCard.css';
+
+//#region Import UI Components
+import IconButton from '../../Buttons/IconButton/IconButton';
+//#endregion
+
+//#region Import Icons
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faScrewdriverWrench } from '@fortawesome/free-solid-svg-icons';
+import { HiMinusSm, HiPlusSm } from 'react-icons/hi';
+//#endregion
+
+// Define SpecifyModelQuantityCard component
+function SpecifyModelQuantityCard(props) {
+
+  // Destructure props
+  const { className, modelId, modelName, modelPhoto, typeName, availableCount, quantity, onIncreaseQuantity, onDecreaseQuantity } = props;
+
+  // State to handle model photo
+  const [equipmentModelPhoto, setEquipmentModelPhoto] = useState(modelPhoto);
+
+  // Singular or plural text based on available count
+  const itemText = availableCount > 1 ? 'items' : 'item';
+
+  // Event handler for increasing quantity
+  const HandleIncreaseQuantity = () => {
+    onIncreaseQuantity(modelId);
+  }
+
+  // Event handler for decreasing quantity
+  const HandleDecreaseQuantity = () => {
+    onDecreaseQuantity(modelId);
+  }
+
+  return (
+    <div className={`SpecifyModelQuantityCard-GeneralContainer  ${className}`}>
+      <div className='SpecifyModelQuantityCard-Container'>
+        <div className='SpecifyModelQuantityCard-ModelPhoto'>
+          {equipmentModelPhoto && (
+            <img src={modelPhoto}
```

Comment:
Aand, same Sonarlint yet again :/

Reply:
Oh yeah, it was because of the OnError function of img and img does not seen as button. However, there seems to not having the other ways to treat it as button so I think we will just go with it for now!